pyteal/ast/abi/util_test.py

```diff
@@ -313,3 +315,150 @@ def test_make():
 
     assert actual.type_spec() == expected_type_spec
     assert type(actual) is abi.Tuple
+
+
+def test_abi_type_translation():
+    test_cases = [
```

Comment:
recreated from [a unit test in the python sdk](https://github.com/algorand/py-algorand-sdk/blob/develop/test_unit.py#L3483).